client/src/components/DetailsCard/index.js

```diff
@@ -1,34 +1,91 @@
 
 import "./detailscard.css";
 
-const DetailsCard = ({ boardingTime, destination, origin, flightNo }) => (
-  <div className="center-div">
-    <div className="container-card-details">
-      <div className="labels-container">
-        <label className="details-label">Flight number</label>
-        <label className="float-style">{flightNo}</label>
-      </div>
-      <div className="labels-container">
-        <label className="details-label">From</label>
-        <label className="float-style">{origin}</label>
-      </div>
-      <div className="labels-container">
-        <label className="details-label">To</label>
-        <label className="float-style">{destination}</label>
-      </div>
-      <div className="labels-container">
-        <label className="details-label">Arrival Time</label>
-        <label className="float-style">{boardingTime}</label>
+class DetailsCard extends Component {
+  getHowManyhourToDepart(mytime) {
+    const dt1 = new Date();
+    const dt2 = new Date(mytime);
+    var diff = (dt2.getTime() - dt1.getTime()) / 1000;
+    diff /= 60 * 60;
+    return Math.abs(Math.round(diff));
+  }
+
+  render() {
+    const {
+      adminPanel,
+      boarding_time,
+      destination,
+      origin,
+      flight_no,
+      detailsTitle,
+      detailsInfoArray
+    } = this.props;
+
+    return (
+      <div>
+        {adminPanel ? (
+          <div className="center-div">
+            <div className="container-card-details">
+              <div className="labels-container">
+                <label className="details-label">Flight number</label>
+                <label className="float-style">{flight_no}</label>
+              </div>
+              <div className="labels-container">
+                <label className="details-label">From</label>
+                <label className="float-style">{origin}</label>
+              </div>
+              <div className="labels-container">
+                <label className="details-label">To</label>
+                <label className="float-style">{destination}</label>
+              </div>
+              <div className="labels-container">
+                <label className="details-label">Arrival Time</label>
+                <label className="float-style">{boarding_time}</label>
+              </div>
+            </div>
+          </div>
+        ) : (
+          <div className="center-div">
+            <div className="customer-container-card-details">
+              <div className="labels-container">
+                <label className="details-title-style">
+                  Flight no : {detailsTitle}
+                </label>
+              </div>
+              <div className="labels-container">
+                <label className="details-label">
+                  Expected to depart in over
+                </label>
+                <label className="float-style">
+                  {this.getHowManyhourToDepart(detailsInfoArray.departure_time)}{" "}
```

Comment:
The momentjs library does a really good job of displaying relative times like this, if you're interested.  Check out https://momentjs.com.

Here are some examples from their page:
```
moment("20111031", "YYYYMMDD").fromNow(); // 7 years ago
moment("20120620", "YYYYMMDD").fromNow(); // 6 years ago
moment().startOf('day').fromNow();        // 10 hours ago
moment().endOf('day').fromNow();          // in 14 hours
moment().startOf('hour').fromNow();       // 4 minutes ago
```